node/external/transactionAPI/errors.go

```diff
@@ -0,0 +1,15 @@
+package transactionAPI
+
+import "errors"
+
+// ErrTransactionNotFound signals that a transaction was not found
+var ErrTransactionNotFound = errors.New("transaction not found")
+
+// ErrCannotRetrieveTransaction signals that a transaction was not found
+var ErrCannotRetrieveTransaction = errors.New("transaction cannot be retrieved")
+
+// ErrNilStatusComputer signals that user account has a nil data trie
```

Comment:
`// ErrNilStatusComputer signals that the status computer is nil`

Reply:
done.